ts/utils/login.ts

```diff
@@ -17,6 +19,25 @@ type LoginFailure = {
 
 type LoginResult = LoginSuccess | LoginFailure;
 
+/**
+ * return some(intentFallbackUrl) if the given input is a valid intent and it has the fallback url
+ * more info https://developer.chrome.com/docs/multidevice/android/intents/
+ * @param intentUrl
+ */
+export const getIntentFallbackUrl = (intentUrl: string): Option<string> => {
+  const intentParse = new URLParse(intentUrl, true).protocol;
```

Comment:
We don't need to parse the URL since we don't use the `query` fields: `new URLParse(intentUrl, false)` might be more appropriate.

Reply:
This is obsolete since we made this change f53fd72dd2db85a2dd27e0ea41e0fb3f440f16d2